h2o-core/src/test/java/water/fvec/FrameTest.java

```diff
@@ -508,4 +509,35 @@ public void testToCSVEscapedQuotes() throws Exception {
     }
   }
 
+  @Test
+  public void testMakeSimilarlyDistributed() {
+    try {
+      Scope.enter();
+      final int[] dataBig = ArrayUtils.seq(0, 1001);
+      final int[] dataSmall = ArrayUtils.seq(0, 231);
+
+      final Frame big = new TestFrameBuilder()
+              .withVecTypes(Vec.T_NUM)
+              .withDataForCol(0, dataBig)
+              .withChunkLayout(10L, 0L, 300L, 400L, 50L, 0, 0, 241, 0)
```

Comment:
Using suffix for long (L) just for half of the arguments seems oddly.

Reply:
ouch! thanks for noticing